web/app.fluidity.money/app/constants.ts

```diff
@@ -0,0 +1,3 @@
+// Not confident that this will work properly in the interim. So I'm providing a default.
+export const GTM_ID = process?.env["GTM_ID"] ?? "GTM-W7QJGR2";
```

Comment:
I don't like the idea of hardcoding it so it's always on, if it's not available the feature should just not enable